### PR TITLE
Adding Archive features

### PR DESCRIPTION
**Pull request by Jogima-cyber**

### CODE REVIEW

File: test/azure.cs [hunk 1]
```diff
@@ -81,6 +86,41 @@ public async Task small_file_azure()
...
+            await store.TryUnArchiveBlobAsync(new Hash("B2EA9F7FCEA831A4A63B213F41A8855B"));
+
+            Boolean finished = false;
+            while (!finished)
+            {
+                UnArchiveStatus status = await store.TryUnArchiveBlobAsync(new Hash("B2EA9F7FCEA831A4A63B213F41A8855B"));
+                if (status == UnArchiveStatus.Done)
+                {
+                    var a2 = store[new Hash("B2EA9F7FCEA831A4A63B213F41A8855B")];
+                    Assert.Equal("B2EA9F7FCEA831A4A63B213F41A8855B", a2.Hash.ToString());
```
Comment (VictorNicollet) [inline test/azure.cs:115]: This will always be true, without even looking at the contents of the remote file (`store[hash]` just stores the hash inside the returned blob without checking anything). 

You need to load the blob's contents and compare them with variable `file` above.

File: test/Lokad.ContentAddr.Azure.Tests.csproj [hunk 2]
```diff
@@ -16,6 +16,16 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
```
Comment (VictorNicollet) [inline test/Lokad.ContentAddr.Azure.Tests.csproj:16]: The changes to this file should not be included in the PR.

File: src/Lokad.ContentAddr.Azure.csproj [hunk 3]
```diff
@@ -23,11 +23,14 @@
```
Comment (VictorNicollet) [inline src/Lokad.ContentAddr.Azure.csproj:23]: The changes to this file should not be included in the PR.

**Review by VictorNicollet — CHANGES_REQUESTED** [review]

File: test/azure.cs (hunk 1 above)
Comment (Jogima-cyber) [reply test/azure.cs]: This mistake should be fixed now.

File: test/Lokad.ContentAddr.Azure.Tests.csproj (hunk 2 above)
Comment (Jogima-cyber) [reply test/Lokad.ContentAddr.Azure.Tests.csproj]: I've committed the initial file.
